dashboard/client/src/pages/layout/index.tsx

```diff
@@ -18,7 +18,6 @@ import { RouteComponentProps } from "react-router-dom";
 import { GlobalContext } from "../../App";
 import { UsageStatsAlert } from "../../common/UsageStatsAlert";
 
-import SpeedTools from "../../components/SpeedTools";
```

Comment:
Should we also delete the SpeedTools.tsx file?

Reply:
Will remove. Can you approve the PR so that I can merge it after removing it over the weekends?!